torch_tvm/managed_tensors.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include <memory>
+#include <unordered_map>
+
+#include <dlpack/dlpack.h>
+#include <tvm/runtime/device_api.h>
+
+#include <torch/csrc/jit/ir.h>
+#include <ATen/DLConvertor.h>
+
+#include "memory_utils.h"
+
+namespace torch_tvm {
+namespace {
+}
+
+using torch::jit::Value;
+
+using torch_tvm::utils::DLManagedTensorPtr;
+
+class ManagedTensors {
+  public:
+    ManagedTensors() = default;
+    bool insert_value(const Value* val) {
```

Comment:
BTW, generally we use `insertValue` kind of naming convention?

Reply:
Yes, removing this whole thing anyway.